src/kpro_batch_v01.erl

```diff
@@ -0,0 +1,247 @@
+%%%   Copyright (c) 2014-2018, Klarna AB
```

Comment:
Nit: new module, so copyright should just be 2018, unless this is (mostly) a copy of older code.